util/internal/integrationtest/common/src/main/java/org/hibernate/search/util/impl/integrationtest/common/rule/BackendWorkThreadingExpectations.java

```diff
@@ -47,7 +47,7 @@ public void awaitIndexingAssertions(ThrowingRunnable assertions) {
 		else {
 			await().pollDelay( Duration.ZERO )
 					.pollInterval( iterative( duration -> duration.multipliedBy( 2 ), Duration.ofMillis( 5 ) ) )
-					.atMost( Duration.ofSeconds( 5 ) )
+					.atMost( Duration.ofSeconds( 15 ) )
```

Comment:
I'm curious, why was this necessary exactly? A specific test?

Reply:
There are some random failing tests. Most of the time: `AutomaticIndexingPolymorphicInverseSideAssociationIT` or `AutomaticIndexingEmbeddedBridgeIT`.

Reply:
That's quite odd.